telegram/_bot.py

```diff
@@ -9248,7 +9280,7 @@ async def replace_sticker_in_set(
         self,
         user_id: int,
         name: str,
-        old_sticker: str,
+        old_sticker: Union[str, Sticker],
```

Comment:
```suggestion
        old_sticker: Union[str, "Sticker"],
```
this is different than the other methods?